components/brave_wallet_ui/components/extension/post-confirmation/index.tsx

```diff
@@ -0,0 +1,11 @@
+import TransactionFailed from './failed'
```

Comment:
Let's not do this anymore. Less files to maintain and less surface area for bugs.